### PR TITLE
Update french translation file of kbrp-sweps

### PR DESCRIPTION
**Pull request by Emilca1**
Improved french translation

### CODE REVIEW

File: kbrp-sweps/sh_language_fr.lua [hunk 1]
```diff
@@ -78,8 +78,8 @@ DarkRP.RegisterSentences("kbrp_sweps", "fr", {
 	["customFrequenceFrameDescription"] = "Votre nouvelle fréquence: %s",
 	["validateButton"] = "Valider",
 	["qteAction§*"] = "[CLIC GAUCHE]",
-	["qteSuccessfulNotification"] = "Vous avez réussi le QTE",
-	["qteFailedNotification"] = "Vous n'avez pas réussi votre QTE",
+	["qteSuccessfulNotification"] = "Vous avez réussi la tentative",
+	["qteFailedNotification"] = "Vous n'avez pas réussi votre tentative",
```
Comment (hugoheml) [inline kbrp-sweps/sh_language_fr.lua:82]: "QTE" must be used here to be more explicit

**Review by hugoheml — CHANGES_REQUESTED** [review]
At first, thanks for your contribution! You just need to edit this two lines with the original text and after, you pull request will be accepted and merged.

File: kbrp-sweps/sh_language_fr.lua (hunk 1 above)
Comment (Emilca1) [reply kbrp-sweps/sh_language_fr.lua:82]: I don't think so. QTE (Quick Time Event) is a complicated term that not everyone knows.
Maybe "test de rapidité" could be more explicit ?

Comment (hugoheml) [reply kbrp-sweps/sh_language_fr.lua:82]: Yes, you can use "test de rapidité" also if you want.
If you want, you can also do a PR in the kbrp-tresors language to resolve this same translation problem.

**Review by hugoheml — APPROVED** [review]
Approved

**Pull request merged**